topostats/grains.py

```diff
@@ -688,10 +642,9 @@ def __init__(
         unet_config: dict[str, str | int | float | tuple[int | None, int, int, int] | None] | None = None,
         threshold_method: str | None = None,
         otsu_threshold_multiplier: float | None = None,
-        threshold_std_dev: dict[str, float | list] | None = None,
-        threshold_absolute: dict[str, float | list] | None = None,
-        area_thresholds: dict[str, list[float | None]] | None = None,
-        direction: str | None = None,
+        threshold_std_dev: list | None = None,
+        threshold_absolute: list | None = None,
+        area_thresholds: list[float | None] | None = None,
```

Comment:
Suggest using a consistent suffix for similar attributes.


```suggestion
        threshold_area: list[float | None] | None = None,
```